met/src/tools/core/ensemble_stat/ensemble_stat_conf_info.cc

```diff
@@ -829,9 +829,24 @@ void EnsembleStatVxOpt::process_config(GrdFileType ftype, Dictionary &fdict,
    // Conf: sid_exc
    vx_pd.set_sid_exc_filt(parse_conf_sid_list(&odict, conf_key_sid_exc));
 
-   // Conf: obs_qty
-   vx_pd.set_obs_qty_filt(parse_conf_obs_qty(&odict));
-
+   // Conf: obs_qty_inc
+   // Check for old "obs_quality" field
+   StringArray obs_qty_sa = odict.lookup_string_array(conf_key_obs_qty);
```

Comment:
All of this logic is already handled in parse_conf_obs_qty_inc() and should be removed from this spot, as you did in point_stat_conf_info.cc.

Reply:
Yes, that was an older version of the file. I modified it. I checked this morning that it was checked in. GitHub indicated it was already checked in. Let me try to re-push it up.

Reply:
I just pushed up the latest version. I don't know why it didn't recognize the change previously.